mocks/juniper/course_modes/models.py

```diff
@@ -0,0 +1,13 @@
+
+
+
+class CourseMode(object):
```

Comment:
Out of scope: It would be nice to have an automatic check if those mock actually exists in Juniper and the "signature" matches correctly.